jjb/ci-management/ci-packer.yaml

```diff
@@ -0,0 +1,22 @@
+- project:
+    name: packer-verify
+    jobs:
+      - gerrit-packer-verify
+
+    project: ci-management
+    project-name: ci-management
+    build-node: centos7-builder-2c-1g
+
+- project:
+    name: packer-builder-jobs
+    jobs:
+      - gerrit-packer-merge
+
+    project: ci-management
+    project-name: ci-management
+    build-node: centos7-builder-2c-1g
+
+    templates: builder
+    platforms:
+      - centos-7
+      - ubuntu-16.04
```

Comment:
We deprecated Ubuntu16
Let's change this to Ubuntu 18.04 instead

Reply:
Fixed